homeassistant/components/niko_home_control/light.py

```diff
@@ -1,49 +1,56 @@
 """Support for Niko Home Control."""
 import logging
-
 import voluptuous as vol
 
-from homeassistant.components.light import (
-    ATTR_BRIGHTNESS, PLATFORM_SCHEMA, Light)
-from homeassistant.const import CONF_HOST
+from datetime import timedelta
```

Comment:
please order your import, https://developers.home-assistant.io/docs/en/development_guidelines.html#ordering-of-imports